frontend/src/styles/main.scss

```diff
@@ -31,13 +31,12 @@ $global-side-padding: 138 * $px;
   box-shadow: 0 0 0 $px $input-border-outlined;
 }
 
-.ednaButton {
+.primaryButton {
```

Comment:
this buttons stuff is moved to a separate component in further branch btw :putya: